test/spy-test.js

```diff
@@ -240,7 +240,26 @@
             }
         },
 
-        call: {
+        // test for regression introduced in #692. See #1046 for details
+        "localStorage support": {
```

Comment:
Maybe this name is a bad idea? It basically says that we support stubbing localStorage, which is impossible in some browsers (like Firefox). I would still like to document the fix, though ...